src/js/component.js

```diff
@@ -134,9 +137,6 @@ class Component {
     this.childIndex_ = null;
     this.childNameIndex_ = null;
 
-    // Remove all event listeners.
-    this.off();
```

Comment:
Sounds good. Just wanted an answer the question before we were to merge it in :)